src/core/content/util.ts

```diff
@@ -703,6 +703,25 @@ export async function getOption(
 	return false;
 }
 
+/**
+ * Read global option from storage.
+ * @param key - Option key
+ * @returns Option value
+ */
+/* istanbul ignore next */
+export async function getGlobalOption(
```

Comment:
@yayuyokitano do we have a function to get at global options? if not would it be worth moving this to `options.ts`?

Reply:
There is an existing `getOption()` function in `options.ts`, but when I tried it out earlier I was running into issues when trying to import and use it - it kept throwing `Uncaught ReferenceError: Cannot access 'OPTIONS' before initialization`.

I've just tried again in a1101ef with an async `import` and it seems to work just fine! 🎉 so I've deleted the new `getGlobalOption()` function I had added. Let me know if you want me to squash those commits together.